test/e2e/flowaggregator_test.go

```diff
@@ -343,6 +348,12 @@ func testHelper(t *testing.T, data *TestData, isIPv6 bool) {
 	// perftest-a (Egress reject) -> perftest-b , perftest-a (Egress drop) -> perftest-d
 	t.Run("IntraNodeDenyConnEgressANP", func(t *testing.T) {
 		skipIfAntreaPolicyDisabled(t)
+		label := "IntraNodeDenyConnEgressANP"
+		// As we utilize the same Perftest Pods for traffic transmission, there's a potential for collecting records from
+		// previous subtests. To mitigate this, we set a label to Perftest Pods during each subtest before initiating
```

Comment:
we add a different label to perftest Pods

Reply:
Fixed.